deploy/olm-catalog/ibm-helm-repo-operator/3.6.2/ibm-helm-repo-operator.v3.6.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,381 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "operator.ibm.com/v1alpha1",
+          "kind": "HelmRepo",
+          "metadata": {
+            "annotations": {
+              "helm.operator-sdk/upgrade-force": "True"
+            },
+            "labels": {
+              "app.kubernetes.io/instance": "helm-repo",
+              "app.kubernetes.io/managed-by": "helmrepos.operator.ibm.com",
+              "app.kubernetes.io/name": "helm-repo"
+            },
+            "name": "helm-repo"
+          },
+          "spec": {
+            "helmrepo": {
+              "resources": {
+                "limits": {
+                  "cpu": "100m",
+                  "memory": "512Mi"
+                },
+                "requests": {
+                  "cpu": "100m",
+                  "memory": "512Mi"
+                }
+              }
+            },
+            "replicaCount": 1
+          }
+        },
+        {
+          "apiVersion":"operator.ibm.com/v1alpha1",
+          "kind":"OperandRequest",
+          "metadata":{
+            "name":"helm-repo-request"
+          },
+          "spec":{
+            "requests":[
+              {
+                "operands":[
+                  {
+                    "name":"ibm-cert-manager-operator"
+                  },
+                  {
+                    "name":"ibm-mongodb-operator"
+                  },
+                  {
+                    "name":"ibm-iam-operator"
+                  },                                    
+                  {
+                    "name":"ibm-management-ingress-operator"
+                  },
+                  {
+                    "name":"ibm-platform-api-operator"
+                  }
+                ],
+                "registry":"common-service"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    categories: Developer Tools
+    certified: "false"
+    containerImage: quay.io/opencloudio/ibm-helm-repo-operator:latest
+    createdAt: "2020-05-16"
+    description: Helm Repo is a helm chart repository for storing and supplying IBM
+      and local charts.
+    olm.skipRange: ">=3.5.0 <3.6.0"
```

Comment:
shouldn't this be `>=3.5.0 <3.6.2`

Reply:
Good catch, looks like we didn't set the skipRange correctly in 3.6.1, that's why the bump-up script didn't update it as well.